inline-c-cpp/test/tests.hs

```diff
@@ -275,16 +297,16 @@ main = Hspec.hspec $ do
 tag :: C.CppException -> String
 tag (C.CppStdException {}) = "CppStdException"
 tag (C.CppHaskellException {}) = "CppHaskellException"
-tag (C.CppOtherException {}) = "CppStdException"
+tag (Legacy.CppOtherException {}) = "CppStdException"
```

Comment:
Shouldn't this be `"CppOtherException"` before and after?

Reply:
This would also have been pointed out by `-Wall` for the test suite.

Fixed in my propsed PR https://github.com/fpco/inline-c/pull/154